pkg/arvo/sys/vane/ames.hoon

```diff
@@ -6091,6 +6091,33 @@
     ::
         [%protocol %version ~]
       ``noun+!>(protocol-version)
+    ::
+        [%boot ~]
+      =/  who
+        =/  ship  our
+        |-
+        ^-  @p
+        =/  next  (^^sein:title rof /eyre our now ship)
```

Comment:
```suggestion
        =/  next  (^^sein:title rof /ames our now ship)
```

This is ames doing the scry, provenance path should reflect that right?

Reply:
Also, this (and the same bit of logic in eyre) will have to change when directed messaging comes out, right? Because then we properly go to the "direct"/"first" sponsor as opposed to always up to the galaxy level?

If that's the case, would be good to have `TODO` comments on these.